Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <label for="secondsinput">Load Time (in seconds)</label>
+    <input id="secondsinput" type="number" min="0" value="3"><br>
```

Comment:
Typically people don't use `<br>` anymore. The recommended way is strictly use CSS to do any kind of visual changes. Makes your markup a lot cleaner and easier to figure out what is causing that blank line.

Reply:
Additionally I believe that the <br> is technically supposed to be written as a self closing tag like:
```
<br />
```
I would agree with @theRingleman in that <br /> element should not generally be used for presentational purposes, unless it is added inside a content block to create a line wrap for specific content types (think poems, addresses, etc)

Something this example:

```
<p>
    Boaty mcBoatface
    20 street name<br />
    City Name<br />
    etc.
</p>
```
That being said, I am the first to admit that I have used <br /> far too often during quick edits on sites, where I should have used a span and css.